test/functional/wallet_backwards_compatibility.py

```diff
@@ -126,6 +127,77 @@ def test_v19_addmultisigaddress(self):
         wallet = node_v19.get_wallet_rpc("w1_v19")
         assert wallet.getaddressinfo(address_18075)["solvable"]
 
+
+    def test_v22_inactivehdchain_path(self):
+        self.log.info("Testing inactive hd chain bad derivation path cleanup")
+        # 0.21.x and 22.x would both produce bad derivation paths when topping up an inactive hd chain
+        # Make sure that this is being detected and automatically cleaned up
+        node_master = self.nodes[1]
+        node_v22 = self.nodes[self.num_nodes - 5]
+        wallet_name = "bad_deriv_path"
+        node_v22.createwallet(wallet_name=wallet_name, descriptors=False)
+        wallet = node_v22.get_wallet_rpc(wallet_name)
```

Comment:
Nit if retouched:

```diff
- wallet = node_v22.get_wallet_rpc(wallet_name)
+ bad_deriv_wallet = node_v22.get_wallet_rpc(wallet_name)
```

For easy identification because this wallet is used across the whole test where other wallets are also created later down.

Reply:
Done